hack/dind-cluster.sh

```diff
@@ -666,8 +689,7 @@ NODE_PREFIX="${CLUSTER_ID}-node-"
 NODE_COUNT=2
 NODE_NAMES=()
 
-BRIDGE_PREFIX="docker"
-BRIDGE_START_NUM=10
+ADDITIONAL_BRIDGE_NAME="${ADDITIONAL_BRIDGE_NAME:-docker10}"
 IPAM_START_NUM=100
```

Comment:
`IPAM_START_NUM` doesn't make sense as a name if we only support one extra network. (`ADDITIONAL_NETWORK_NUMBER`?) Also, 172.100.0.0/16 isn't a private-use network (only 172.16.0.0/16 - 172.31.0.0/16 are reserved). Maybe use 18 instead since it comes right after the default docker0 range? And do a `ADDITIONAL_NETWORK_NUMBER="${ADDITIONAL_NETWORK_NUMBER:-18}"` like the other variables so the caller can override it.